deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -2,6 +2,57 @@
 
 readonly DEBUG_MODE=${DEBUG_MODE:="false"}
 readonly DEBUG_MODE_ENABLED_FLAG="true"
+readonly TEST_PVC_FILE="test-pvc-ss.yaml"
+readonly TEST_PVC_STATEFULSET="pvc-nginx"
+readonly TEST_PVC_STATEFULSET_CONFIG="apiVersion: v1
```

Comment:
Is it possible to move it to config map?

Reply:
Most env variables are in `job.yaml`, wouldn't it be better to move it there?

Reply:
I was referring to `TEST_PVC_STATEFULSET_CONFIG` :sweat_smile:

Reply:
I'll have to check how to access it in the setup image.

Reply:
I don't think we need it in image, we can simply use configmap to add it in runtime: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/templates/setup/configmap.yaml

In dockerfile, we only get files required for `terraform init` to have dependencies already in image